### PR TITLE
Longest Common Subsequence in C++

### PR DESCRIPTION
**Pull request by dibyendu415**

### CODE REVIEW

File: Task 3/Longest Common Subsequence/Longest Common Subsequence.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#include<bits/stdc++.h>
```
Comment (nimishbongale) [inline Task 3/Longest Common Subsequence/Longest Common Subsequence.cpp:1]: Can we have a multiline comment at the beginning of this file having a short explanation of the problem with sample input/output cases?

**Review by nimishbongale — COMMENTED** [review]
Good work @dibyendu415!

Just a small change. Let's get this merged!

File: Task 3/Longest Common Subsequence/Longest Common Subsequence.cpp (hunk 1 above)
Comment (dibyendu415) [reply Task 3/Longest Common Subsequence/Longest Common Subsequence.cpp:7]: @nimishbongale Explanation added

**Review by nimishbongale — APPROVED** [review]
Great work @dibyendu415! 🚀 

You are now ready to contribute to cool open source projects! 😄

**Pull request merged**